content/general/concepts/algorithm/terms/euclidean-algorithm/euclidean-algorithm.md

```diff
@@ -0,0 +1,149 @@
+---
+Title: 'Euclidean algorithm'
+Description: 'The Euclidean algorithm is a simple and efficient method for finding the highest common factor (HCF), also known as the greatest common divisor (GCD), of two numbers.'
```

Comment:
```suggestion
Description: 'A simple and efficient method for finding the highest common factor (HCF), also known as the greatest common divisor (GCD), of two numbers.'
```
This because of the way terms are displayed on the parent page. They show up as [title] - [description] so having the title of the entry in the description will cause a redundancy.

Reply:
alright